server/services/random.js

```diff
@@ -1,3 +1,6 @@
+const minResourceScore = 10;
+const maxResourceScore = 50;
```

Comment:
constants should be written in CAPITAL_LETTERS_WITH_UNDERCASE

Reply:
These look like placeholder values. Remove them and instead use parameters on the function.